modules/operator/main.tf

```diff
@@ -108,11 +129,20 @@ resource "kubectl_manifest" "falcon_container_sensor" {
   ]
 }
 
-# # Deploy admission controller if var.falcon_admission = true
+# Deploy admission controller if var.falcon_admission = true
 resource "kubectl_manifest" "falcon_admission_controller" {
   count     = var.falcon_admission ? 1 : 0
   yaml_body = var.admission_controller_manifest_path == "default" ? local.default_admission_controller_manifest : data.local_file.admission_controller_manifest[0].content
   depends_on = [
     kubectl_manifest.falcon_operator
   ]
 }
+
+# Deploy admission controller if var.iar = true
+resource "kubectl_manifest" "iar" {
+  count     = var.iar ? 1 : 0
+  yaml_body = var.iar_manifest_path == "default" ? local.default_iar_manifest : data.local_file.iar_manifest[0].content
+  depends_on = [
+    kubectl_manifest.falcon_operator
+  ]
+}
```

Comment:
File is missing a trailing newline character.

Reply:
ack